site/content/en/docs/Installation/helm.md

```diff
@@ -36,8 +36,12 @@ _We recommend to install Agones in its own namespaces (like `agones-system` as s
 you can use the helm `--namespace` parameter to specify a different namespace._
 
 When running in production, Agones should be scheduled on a dedicated pool of nodes, distinct from where Game Servers are scheduled for better isolation and resiliency. By default Agones prefers to be scheduled on nodes labeled with `agones.dev/agones-system=true` and tolerates node taint `agones.dev/agones-system=true:NoExecute`. If no dedicated nodes are available, Agones will
+<<<<<<< HEAD
```

Comment:
Need to resolve this merge conflict.

Reply:
Should be resolved.

Reply:
@roberthbailey is this good to go now?

(I'm on a push to close out PRs before EOY :smiley:)